README.md

```diff
@@ -83,9 +57,72 @@ $ cargo build --locked
 ```
 $ cargo test --locked
 ```
+## Usage
+
+First, generate a RISC-U binary (full support of RISC-V is coming soon!). Refer to the [selfie repositorie](https://github.com/cksystemsteaching/selfie). Once selfie is installed you can generate a binary for a code you have written in a file (e.g `hello.c`):
```

Comment:
typo: "repository"
nitpick: I would write "[...] for code you have written [...]" and drop the "a"